src/rules/no_const_assign.rs

```diff
@@ -210,25 +226,70 @@ mod tests {
 
   #[test]
   fn no_const_assign_invalid() {
-    assert_lint_err::<NoConstAssign>("const x = 0; x = 1;", 13);
-    assert_lint_err::<NoConstAssign>("const {a: x} = {a: 0}; x = 1;", 23);
-    assert_lint_err::<NoConstAssign>("const x = 0; ({x} = {x: 1});", 15);
-    assert_lint_err::<NoConstAssign>("const x = 0; ({a: x = 1} = {});", 14);
-    assert_lint_err::<NoConstAssign>("const x = 0; x += 1;", 13);
-    assert_lint_err::<NoConstAssign>("const x = 0; ++x;", 13);
-    assert_lint_err::<NoConstAssign>(
-      "const x = 0; function foo() { x = x + 1; }",
-      30,
-    );
-    assert_lint_err::<NoConstAssign>(
-      "const x = 0; function foo(a) { x = a; }",
-      31,
-    );
-    assert_lint_err::<NoConstAssign>("for (const i = 0; i < 10; ++i) {}", 26);
-    assert_lint_err::<NoConstAssign>(
-      "const x = 0; while (true) { x = x + 1; }",
-      28,
-    );
+    assert_lint_err! {
+      NoConstAssign,
+      r#"const x = 0; x = 1;"#: [
+      {
+        col:  13,
```

Comment:
```suggestion
        col: 13,
```

Very small nit: extra whitespace here
(same for the other test cases)